includes/admin/class-amp-editor-blocks.php

```diff
@@ -204,4 +212,110 @@ public function print_dirty_amp_scripts() {
 			wp_scripts()->do_items( $this->content_required_amp_scripts );
 		}
 	}
+
+	/**
+	 * Registers the dynamic block Latest Stories.
+	 * Much of this is taken from the Core block Latest Posts.
+	 *
+	 * @see register_block_core_latest_posts()
+	 */
+	public function register_block_latest_stories() {
+		register_block_type(
+			'amp/amp-latest-stories',
+			array(
+				'attributes'      => array(
+					'className'     => array(
+						'type' => 'string',
+					),
+					'storiesToShow' => array(
+						'type'    => 'number',
+						'default' => 5,
+					),
+					'order'         => array(
+						'type'    => 'string',
+						'default' => 'desc',
+					),
+					'orderBy'       => array(
+						'type'    => 'string',
+						'default' => 'date',
+					),
+				),
+				'render_callback' => array( $this, 'render_block_latest_stories' ),
+			)
+		);
+	}
+
+	/**
+	 * Renders the dynamic block Latest Stories.
+	 * Much of this is taken from the Core block Latest Posts.
+	 *
+	 * @see render_block_core_latest_posts()
+	 * @param array $attributes The block attributes.
+	 * @return string $markup The rendered block markup.
+	 */
+	public function render_block_latest_stories( $attributes ) {
+		$args = array(
+			'post_type'        => AMP_Story_Post_Type::POST_TYPE_SLUG,
+			'posts_per_page'   => $attributes['storiesToShow'],
+			'post_status'      => 'publish',
+			'order'            => $attributes['order'],
+			'orderby'          => $attributes['orderBy'],
+			'suppress_filters' => false,
+		);
+
+		$story_query = new WP_Query( $args );
+
+		ob_start();
+		foreach ( $story_query->posts as $post ) :
+			?>
+			<a href="<?php echo esc_url( get_permalink( $post ) ); ?>">
+				<?php echo get_the_post_thumbnail( $post->ID, self::LATEST_STORIES_IMAGE_SIZE ); ?>
```

Comment:
I think the underlying title of the story should be included here, perhaps as the `alt` text or some other text that is displayed alongside the image.

Reply:
Sure, 843f993e3167949ad268b2f1b73d0aae09a17091 adds the story title as the `img[alt]`.

I'm also looking for a way to display the title.

Reply:
The current title display is [shown below](https://github.com/ampproject/amp-wp/pull/1916#issuecomment-469291561).